contracts/oracles/BoundValidator.sol

```diff
@@ -18,14 +18,18 @@ struct ValidateConfig {
 
 // BoundValidator provides some common functions and can be used
 // to wrap up other contracts to form pivot oracles
-contract BoundValidator is OwnableUpgradeable, BoundValidatorInterface {
+contract BoundValidator is AccessControlled, BoundValidatorInterface {
     /// @notice validation configs by asset
     mapping(address => ValidateConfig) public validateConfigs;
 
     /// @notice vBNB address
     /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
     address public immutable vBnb;
 
+    /// @notice VAI address
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    address public immutable vai;
```

Comment:
We are not using the `vai` variable in this contract. Should we take it into account in `validatePriceWithAnchorPrice`?

Reply:
Fixed